City-of-Helsinki/drupal-hdbt
# UHF-2939: UHF-2939 Added workaround for the issue that accordion icon…

…s remain open on firefox

# Accordion arrow fix for Firefox [UHF-2939](https://helsinkisolutionoffice.atlassian.net/browse/UHF-2939)
Fix for accordion arrow remaining still after clicking accordion open once on Firefox

## What was done
* THIS IS NOT A PERFECT FIX.
* There is a thread on handorgel about this issue but there is no movement there and the fix proposed there doesn't work in our case since the handorgel__header--open is never removed from Firefox like it should be between clicks. https://github.com/oncode/handorgel
* This causes small flashing when the icon rotates. Very minor but noticable.
* Basically changed the rotate to be tied to handorgel__header--opened class instead of handorgel__header--open since we don't have situation where the accordion header would be open by default where we would need this handorgel__header--open  class. 

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-2939_firefox_bug_workaround_for_accordion`
* Run `make drush-cr`

## How to test
* On any instance just create a accordion and go to Firefox and open the title and close it again. The icon should not remain pointing down anymore like it did before.

## Designers review

One of the checkboxes below needs to be checked (`[x]`)
